diff -up open5gs-2.7.5/subprojects/freeDiameter/libfdcore/meson.build.foo open5gs-2.7.5/subprojects/freeDiameter/libfdcore/meson.build
--- open5gs-2.7.5/subprojects/freeDiameter/libfdcore/meson.build.foo	2025-07-24 07:40:38.288115912 -0500
+++ open5gs-2.7.5/subprojects/freeDiameter/libfdcore/meson.build	2025-07-24 07:41:43.920505657 -0500
@@ -62,7 +62,7 @@ pgen = generator(bison,
 
 pfiles = pgen.process('fdd.y')
 
-libfdcore = library('fdcore',
+libfdcore = library('ogsfdcore',
     sources : [lfiles, pfiles, fdcore_sources],
     version : freeDiameter_version,
     soversion : freeDiameter_soversion,
diff -up open5gs-2.7.5/subprojects/freeDiameter/libfdproto/meson.build.foo open5gs-2.7.5/subprojects/freeDiameter/libfdproto/meson.build
--- open5gs-2.7.5/subprojects/freeDiameter/libfdproto/meson.build.foo	2025-07-24 07:43:12.214393938 -0500
+++ open5gs-2.7.5/subprojects/freeDiameter/libfdproto/meson.build	2025-07-24 07:43:20.849057077 -0500
@@ -33,7 +33,7 @@ fdproto_sources = files('''
     version.c
 '''.split())
 
-libfdproto = library('fdproto',
+libfdproto = library('ogsfdproto',
     sources : [fdproto_sources],
     version : freeDiameter_version,
     soversion : freeDiameter_soversion,
